src/otelcollector/otelcol-config-extras.yml

```diff
@@ -53,10 +53,11 @@ processors:
       - delete_key(attributes, "project_id")
 
   # See https://github.com/open-telemetry/opentelemetry-demo/issues/1330
-  filter/currency:
+  # flagd metrics are written too often.
```

Comment:
nit: This comment seems to indicate that currency metrics are included here for different reason. Is currency metric also written too often?

Reply:
Yes, that is captured in the comment above it.  I plan to fix both soon using the interval processor, so i'm going to leave as-is